### PR TITLE
Update the setup-openebs.yaml to remove the openebs cast deployment role.

### PR DESCRIPTION
**Pull request by dargasudarshan**
Signed-off-by: Sudarshan <sudarshan.darga@cloudbyte.com>

**What this PR does / why we need it**:
- Remove openebs cast deployment role from setup-openebs.yaml
- Updated the openebs cleanup role to remove cast de-provisioning and verification tasks.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml [hunk 1]
```diff
@@ -26,21 +26,6 @@
   delay: 10
   retries: 6
 
-- name: Delete the openebs pre-release features
-  shell: source ~/.profile; kubectl delete -f {{ ansible_env.HOME }}/{{ openebs_pre_release_alias }}
-  args:
-    executable: /bin/bash
-  ignore_errors: yes
-
-- name: Confirm pre-release features have been deleted
```
Comment (kmova) [inline e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: cast .. clean up check should be there. As part of openebs operator clean up, when openebs namespace is deleted, the cast yamls / objects should be deleted.

Comment (dargasudarshan) [reply e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: Yes @kmova Added this cleanup check task post openebs-operator deletion.

Comment (dargasudarshan) [reply e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: Yes @kmova Added this cleanup check task post openebs-operator deletion.

File: e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml [hunk 2]
```diff
@@ -56,11 +41,19 @@
   delay: 30
   retries: 10
 
+- name: Confirm pre-release features have been deleted	
+  shell: source ~/.profile; kubectl get cast -n openebs
```
Comment (gprasath) [inline e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml]: use {{ operator_ns }}

File: e2e/ansible/roles/k8s-openebs-operator/tasks/main.yml [hunk 3]
```diff
@@ -114,6 +114,24 @@
   register: result_vers
   failed_when: "'m-apiserver status:  running' not in result_vers.stdout"
 
+- name: Confirm CAS Templates are deployed
+  shell: source ~/.profile; kubectl get cast -n openebs
```
Comment (gprasath) [inline e2e/ansible/roles/k8s-openebs-operator/tasks/main.yml]: Use variable for namespace

**Pull request merged**